insights/parsers/system_time.py

```diff
@@ -3,7 +3,26 @@
 =========================
 
 This is a collection of parsers that all deal with the system's configuration
-of its time setting resources.
+of its time setting resources:
+
+ChronyConf - file ``/etc/chrony.conf``
+--------------------------------------
+
+ChronydService - file ``/etc/sysconfig/chronyd``
+------------------------------------------------
+
+LocalTime - command ``file -L /etc/localtime``
+----------------------------------------------
+
+NTP_conf - file ``/etc/ntp.conf``
+---------------------------------
+
+NtpTime - command ``ntptime``
+-----------------------------
+
+NTPDService - file ``/etc/sysconfig/ntpd``
+------------------------------------------
+
```

Comment:
Hi @bfahr - I've updated the documentation to move the subheadings into the .rst directly - let me know what you think!